### PR TITLE
OPA policy compatible with istio-1.14.1

### PR DESCRIPTION
**Pull request by deepak7093**
- Update ext-authz EnvoyFilter version and schema

### CODE REVIEW

File: external-auth/manifests/selkies-opa.yaml [hunk 1]
```diff
@@ -172,3 +172,20 @@ spec:
         image: openpolicyagent/kube-mgmt:0.12.1
         args:
           - --enable-data
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app.kubernetes.io/name: broker
+  name: deny-all
```
Comment (danisla) [inline external-auth/manifests/selkies-opa.yaml]: This name should be something like `allow-istio`

**Review by danisla — CHANGES_REQUESTED** [review]

**Comment by danisla** [comment]
@deepak7093 Also, update the opa Service name to `http2-opa-istio`

File: external-auth/manifests/selkies-opa.yaml (hunk 1 above)
Comment (deepak7093) [reply external-auth/manifests/selkies-opa.yaml]: Done

**Comment by deepak7093** [comment]
Changes included in 1.22 upgrade

**Pull request closed without merging**